conf/app.routes

```diff
@@ -1,3 +1,5 @@
 # microservice specific routes
 
 GET        /hello-world             uk.gov.hmrc.pushpullnotificationsgateway.controllers.MicroserviceHelloWorldController.hello
```

Comment:
Maybe we could get rid of hello world now that we have a proper endpoint.

Reply:
I think it needs to remain until this PR has gone through testing so we have an endpoint we know works and can be reached. If the new endpoint is not able to respond or be reached we have a way of checking its not the deployment without scouring the logs